src/components/ont/OntTube.vue

```diff
@@ -16,6 +16,9 @@
 
 <script>
 
+const img = new Image()
+img.src = '/tube.png'
+
```

Comment:
What did changing the scope of this constant achieve?  Making it truly constant?

Reply:
I noticed that if I didn't call `setDragImage` the drag was working as expected in Safari. After some digging I found [this](https://stackoverflow.com/a/22610927) slightly outdated StackOverflow, which did the trick.

I also noticed that if you kept defining the image in the event handler, on Safari it got added as a Source file each time (not sure if just a reference to the same image, a browser cache, or something else).

Pre-defining it and increasing it's scope fixed both of these issues

Reply:
As per the context of the change: the drag wasn't working in Safari

Reply:
Strangely enough it was already like this in pacbio tube. As we are doing the same thing.

Reply:
Could've saved myself some time if I'd looked at that 😆

Reply:
My fault. Should have pointed you to it. I assumed that the same drag and drop had been used. We need to abstract it into the same place as they do pretty much exactly the same thing.